### ISSUE TITLE
Saving where you parked

### ISSUE THREAD

**Issue by limone-eth** [ParkIdle/ParkIdle-Android#9]
- Feature name: Where did you park your car?
- Description: A map in which one could locate where he/she parked.
- Requires: working with maps api, adding client side support
- Due: Milestone 2 (BETA)
- Status: todo.
- Time Estimation: 1 hour
